x-pack/plugins/observability/public/pages/slos/components/auto_refresh_button.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { EuiButtonEmpty } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+interface Props {
+  enabled: boolean;
+  dataTestSubj?: string;
+  disabled?: boolean;
```

Comment:
🍰 nit: `enabled` and `disabled` are not straightforward to understand without looking at the code. Maybe `isAutoRefreshing` instead of enabled?

Reply:
Good point